Jenkinsfile

```diff
@@ -53,27 +52,23 @@ pipeline {
           dir("tools_released") {
             sh "echo ${params.TOOLS_VERSION} > REQUIRED_TOOLS_VERSION"
           }
-          withEnv(["REPO=lib_adat"]) {
-            xcoreLibraryChecks("lib_adat", false)
+          withEnv(["REPO=${REPO}"]) {
+            xcoreLibraryChecks("${REPO}", false)
           }
         }
       }
     }  // Library checks
     stage('Build examples') {
       steps {
         withTools(params.TOOLS_VERSION) {
-          withEnv(["XMOS_CMAKE_PATH=${WORKSPACE}/xcommon_cmake"]) {
-            dir("lib_adat/examples") {
-              script {
-                // Build all apps in the examples directory
-                def apps = sh(script: "ls -d app_*", returnStdout: true).trim()
-                for(String app : apps.split()) {
-                  // First build using XCommon with xmake, then using XCommon CMake
-                  sh "xmake -C ${app}"
-                  sh "xmake -C ${app} clean"
-                  sh "cmake -S ${app} -B ${app}/build -G\"Unix Makefiles\""
-                  sh "xmake -C ${app}/build"
-                }
+          dir("${REPO}/examples") {
+            script {
+              // Build all apps in the examples directory
+              def apps = sh(script: "ls -d app_*", returnStdout: true).trim()
```

Comment:
You could run cmake and then xmake from the examples directory instead of looping through each one.